upgrade_testing/data/upgrade

```diff
@@ -90,10 +90,21 @@ function main() {
         post_tests
         STATUS=$?
     fi
+    check_no_apt_errors
+    STATUS=$?
 
     exit $STATUS
 }
 
+function check_no_apt_errors() {
+    upgrade_log "Checking that the running system has a healthy apt state"
+    if [ apt-get check ]; then
+        upgrade_log "apt is in a healthy state!"
+    else
+        upgrade_log "apt is not in a healthy state!"
```

Comment:
Need to return 1 here.

Reply:
I think we want `STATUS=$?` or something like that to make it behave like the other functions.